### PR TITLE
remove the keymetrics banner when pm2 starts

### PR DESCRIPTION
**Pull request by j33f**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,5 +7,6 @@ RUN set -ex && \
     apk add \
       build-base && \
     npm install && \
+    echo "" > /var/app/node_modules/pm2/lib/keymetrics && \
```
Comment (scottinet) [inline Dockerfile]: I disagree with this way of removing the Keymetrics banner by modifying an external component's files.

I think it's cleaner to simply launch `pm2 start` with the ``--silent`` option (just tested it, it removes the banner and other unnecessary output).

Comment (scottinet) [inline Dockerfile]: (should also be changed for the main Kuzzle project)

Comment (j33f) [inline Dockerfile]: good catch ! missed the --silent option... needed a nap

**Comment by j33f** [comment]
bad idea

**Pull request closed without merging**